src/EventListener/AddLibrarySubscriber.php

```diff
@@ -53,14 +52,13 @@ public function addLibrary(FilterResponseEvent $event): void
             return;
         }
 
-        // we don't want to add the library code on redirects
-        $statusCode = $event->getResponse()->getStatusCode();
-        if (($statusCode < 200 || $statusCode > 299) && !in_array($statusCode, [404, 500], true)) {
-            return;
-        }
+        $this->tagBag->add(new ScriptTag('var _awev=window._awev||[];', Tags::TAG_EVENT_HANDLER), TagBagInterface::SECTION_HEAD);
 
-        $this->tagBag->add(new TwigTag('@SetonoSyliusAddwishPlugin/Tag/library.js.twig', TagInterface::TYPE_SCRIPT, [
-            'partner_id' => $this->partnerId,
-        ]), TagBagInterface::SECTION_BODY_BEGIN);
+        $this->tagBag->add(new TwigTag(
+            '@SetonoSyliusAddwishPlugin/Tag/library.html.twig',
+            TagInterface::TYPE_HTML,
+            Tags::TAG_LIBRARY,
+            ['partner_id' => $this->partnerId]
+        ), TagBagInterface::SECTION_BODY_END);
```

Comment:
I would like to put at body begin to be sure it included earlier than other scripts (from other event handlers), but as far as this work as is - its ok :)

Reply:
It should work asynchronously, so best to put it at the end :)